services/movie.py

```diff
@@ -28,13 +33,14 @@ def create_movie(
     genres_ids: list = None,
     actors_ids: list = None,
```

Comment:
Add type hints for `genres_ids` and `actors_ids` parameters, e.g., `list[int]`, to improve code clarity and type checking.